recipes-core/custom-nftables-rules/files/nftables-maintenance.conf

```diff
@@ -0,0 +1,34 @@
+#!/usr/sbin/nft -f
+
+flush ruleset
+
+# Define our endpoints
+define BUILDER_ENDPOINT = 0.0.0.0     # Replace with actual builder endpoint IP
+
+table inet filter {
+    chain input {
```

Comment:
Should we not accept by default and drop everything coming from & going to `BUILDER_ENDPOINT`?